.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
 
 install:
-- pip install molecule ansible docker-py
+- pip install molecule ansible docker
```

Comment:
Are you sure? [The Molecule docs](https://molecule.readthedocs.io/en/latest/configuration.html#docker) still say to use `docker-py`.